arex-instrumentation/database/arex-database-common/src/main/java/io/arex/inst/database/common/DatabaseHelper.java

```diff
@@ -32,29 +32,29 @@ public static String getDbName(Connection connection) {
                 LogUtil.warn("parseConnection failed, " + url);
                 return null;
             }
-            return getDbName(url, connection.getClientInfo());
+            return getDbName(url, connection);
         } catch (Exception e) {
             LogUtil.warn("parseConnection", e);
         }
         return null;
     }
 
-    public static String getDbName(String url, Properties props) {
-        if (StringUtil.isEmpty(url) || !url.startsWith("jdbc:")) {
-            LogUtil.warn("parseConnection, " + url);
-            return null;
-        }
+    public static String getDbName(String url, Connection connection) {
+        try {
+            String jdbcUrl = url.substring(5);
```

Comment:
1. Make sure the lenght of `url`  what you want before using `substring(5)`
2. Remove the magic number  `5`